README.md

```diff
@@ -60,6 +60,10 @@ cannot open 'zroot/iocell/jails/XXXXXXXX-XXXX-XXXX-a5be-3417ebb13845': dataset d
 
 **QUICK HOWTO:**
 
+Activate your zfs pool:
+
+`iocell activate zroot`
```

Comment:
You might want to add some comment for less experienced users, that their pool name might be different from zroot, depending on their OS installation and/or the fact if they have a separate pool for iocell purposes.